.azuredevops/deploy-platform-bootstrap-serviceconnections.yaml

```diff
@@ -12,7 +12,7 @@ resources:
       name: DEFRA/adp-pipeline-common
       endpoint: DEFRA
       type: github
-      ref: main 
+      ref: gg/214939-sc-accesstoken #main 
```

Comment:
Update this to main when dependent [PR ](https://dev.azure.com/defragovuk/DEFRA-DEVOPS-COMMON/_git/Defra.Pipeline.Common/pullrequest/36783) is completed and merged to main